build/params_mainnet.go

```diff
@@ -23,6 +21,7 @@ var DrandSchedule = map[abi.ChainEpoch]DrandEnum{
 }
 
 var NetworkBundle = "mainnet"
+var BundleOverrides map[actors.Version]string
```

Comment:
@jennijuju is concerned about users misusing this and losing consensus -- can we add a comment on this saying "DON'T SET THIS unless you know what you're doing, you will lose consensus with the network!"